src/vs/workbench/browser/parts/titlebar/titlebarPart.ts

```diff
@@ -310,6 +310,7 @@ export class TitlebarPart extends Part implements ITitleService {
 		const activeFolderMedium = editorFolderResource ? this.labelService.getUriLabel(editorFolderResource, { relative: true }) : '';
 		const activeFolderLong = editorFolderResource ? this.labelService.getUriLabel(editorFolderResource) : '';
 		const rootName = this.labelService.getWorkspaceLabel(workspace);
+		const rootNameShort = this.labelService.getWorkspaceLabel(workspace, { verbose: false, short: true });
```

Comment:
This deserves a test for example in here:

https://github.com/microsoft/vscode/blob/19f6f3103606da3fe5d8be4b722d96fe8e18271a/src/vs/base/test/common/labels.test.ts#L105